app/styles/components/_play-button.scss

```diff
@@ -209,4 +209,4 @@
       opacity: 0;
     }
   }
-}
+}
```

Comment:
this looks like an accidental formatting change. you can get rid of this change so `_play-button.scss` isn't included in the PR.